src/zernike_polynomial.cpp

```diff
@@ -61,16 +75,30 @@ double ZernikePolynomials::evaluate_zernike_at_order(
         "the providede order for the zernike-polynomial is out of the stored "
         "max-order.");
   }
+
+  // get the start and end location of desired order's coefficients
+  std::size_t it_coeff_start = 0;
+  std::size_t it_coeff_end = 0;
+  if (order != 0) {
+    for (std::size_t ord = 0; ord < order; ord++) {
+      std::pair<int, int> n_and_l = get_n_and_l(ord);
+      const std::size_t n = static_cast<std::size_t>(n_and_l.first);
+      const std::size_t m = static_cast<std::size_t>(std::abs(n_and_l.second));
```

Comment:
Why are we changing the variable name from l to m here ? This is very confusing.